src/main/java/thestonedturtle/mahoganyhomes/MahoganyHomesOverlay.java

```diff
@@ -92,17 +96,17 @@ void addMaterialLine(String type, int min, int max, int have)
 		{
 			if (have >= max)
 			{
-				line = line.right("✓").rightColor(Color.GREEN);
+				line = addLineSuffix(line, UNICODE_CHECK_MARK, Color.GREEN);
 			}
 			else if (have >= min)
 			{
 				// Player has between max and min, so it's not
 				// certain if they have enough.
-				line = line.right("?").rightColor(Color.YELLOW);
+				line = addLineSuffix(line, "?", Color.YELLOW);
 			}
 			else
 			{
-				line = line.right("x").rightColor(Color.RED);
+				line = addLineSuffix(line, UNICODE_BALLOT_X, Color.RED);
```

Comment:
is there a reason we can't just do `line.right(UNICODE_BALLOT_X, Color.red);` instead of creating this new `addLineSuffix` function?

Is `FontManager.getDefaultFont()` required for this to work properly?

Reply:
Okay, I wasn't sure if `FontManager.getDefaultFont()` was required for this to work.

I think what you suggested is more readable as regardless of the text we'd want to ensure it renders properly. In the issue even the question mark was rendering weirdly.

Reply:
I'll adjust the Pull Request in a few moments then with the more readable approach. 😄